fe/fe-core/src/main/java/com/starrocks/sql/ast/ShowAnalyzeMetaStmt.java

```diff
@@ -0,0 +1,63 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+package com.starrocks.sql.ast;
+
+import com.google.common.collect.Lists;
+import com.starrocks.analysis.RedirectStatus;
+import com.starrocks.analysis.ShowStmt;
+import com.starrocks.catalog.Column;
+import com.starrocks.catalog.Database;
+import com.starrocks.catalog.ScalarType;
+import com.starrocks.catalog.Table;
+import com.starrocks.cluster.ClusterNamespace;
+import com.starrocks.common.MetaNotFoundException;
+import com.starrocks.qe.ShowResultSetMetaData;
+import com.starrocks.server.GlobalStateMgr;
+import com.starrocks.statistic.AnalyzeMeta;
+
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+public class ShowAnalyzeMetaStmt extends ShowStmt {
+
+    private static final ShowResultSetMetaData META_DATA =
+            ShowResultSetMetaData.builder()
+                    .addColumn(new Column("Database", ScalarType.createVarchar(60)))
+                    .addColumn(new Column("Table", ScalarType.createVarchar(60)))
+                    .addColumn(new Column("Type", ScalarType.createVarchar(20)))
+                    .addColumn(new Column("UpdateTime", ScalarType.createVarchar(60)))
+                    .addColumn(new Column("Healthy", ScalarType.createVarchar(5)))
```

Comment:
what's the meaning of `Healthy `

Reply:
The logic of this code has been written in the comments.
/*

health = totalLoadRows / totalRowCount.
The ratio of the number of modified lines to the total number of lines.
Because we cannot obtain complete table-level information, we use the row count of
the partition with the smallest row count as totalRowCount.
It can be understood that we assume an extreme case where all imported and modified lines
are concentrated in only one partition
*/